examples/google-vertex-ai/README.md

```diff
@@ -0,0 +1,103 @@
+# ChatDKG example - Extractive question answering with OriginTrail and Google Vertex AI
+
+Hello there! This example will show you how to create a basic Extractive question answering system using OriginTrail Knowledge Assets and the Google Vertex AI. Before running the code, we have to create an index with the Vertex AI Matching Engine and deploy a public endpoint that we will query.
+
+With this example you will:
+
+- create an OriginTrail Knowledge Asset from a JSON object, using the DKG JS client
+- create index entries from that Knowledge Asset and upload them to a Vertex AI index
+- execute similarity search over the index via a public endpoint and discover the source Knowledge Assets
+- use Google Chirp speech recognition model to transcribe audio files
+
+In contrast to generative QA systems such as ChatGPT, an extractive system doesn't "hallucinate", rather only extracts content from within the verifiable Knowledge Asset.
+Additionally, to extend the extractive approach, we also demonstrate an "extract & summarize" approach that takes the extracted content from the Knowledge Asset and submits it to an LLM (in this case Vertex AI Prediction Service) to summarize.
+
+## Pre-requisites
+
+- NodeJS 16
+- Access to an OriginTrail DKG node. You can setup your own by following instructions [here](https://docs.origintrail.io/decentralized-knowledge-graph-layer-2/testnet-node-setup-instructions/setup-instructions-dockerless)
+- An account and a project set up on [Google Vertex AI](https://cloud.google.com/vertex-ai).
+- Optionally: OpenAI API key
```

Comment:
@brkagithub Can you remove this please ?

Reply:
you mean only the OpenAI key or the whole thing? OpenAI ended up there accidentally

Reply:
OpenAI key part